src/lookup.rs

```diff
@@ -50,6 +52,31 @@ pub(super) struct ChannelDelta {
 	pub(super) requires_reminder: bool,
 }
 
+pub(super) struct NodeDelta {
+	/// The most recently received, but new-to-the-client, node details
+	pub(super) latest_details_after_seen: Option<NodeDetails>,
+
+	/// Between the last details prior to the `seen` value and now, including any potential
+	/// intermediate updates that are not kept track of here, has the set of features this node
+	/// supports changed?
```

Comment:
The appositive seem superfluous.

Reply:
Ah, given you're explanation to me offline about intermediate updates, this is fine to keep